apps/prairielearn/src/server.js

```diff
@@ -389,27 +389,51 @@ module.exports.initExpress = function () {
   // us to treat those files as immutable and cache them essentially forever.
   app.use(
     '/cacheable_node_modules/:cachebuster',
-    express.static(path.join(__dirname, 'node_modules'), {
+    express.static(path.resolve(__dirname, '..', 'node_modules'), {
```

Comment:
TODO: dry this up with an abstraction that can handle the need to serve from two different `node_modules` directories.

Reply:
This is being done in #7666.